packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -67,6 +69,27 @@ export type ChatCompositeOptions = {
    * Set focus on the composite when the composite first mounts.
    */
   autoFocus?: 'sendBoxTextField' | false;
+  /* @conditional-compile-remove-from(stable) */
```

Comment:
These three arguments are clearly related. I'd vote for creating a nested object to keep them together. Yes JS interfaces seem to have a gazillion fields distinguished with (concatenated) prefixes. No it is not a good thing.

Reply:
Good idea.

Reply:
Changed. LMKWYT